source/references/modules/animation.md

```diff
@@ -47,62 +55,36 @@ Smooth and meaningful animation is very effective for enhancing the user experie
+- `duration` (number): Specifies the number of milliseconds of animation execution, the default value is `0`, means that no animation will occur.    
+- `delay` (number): Specifies the amount of milliseconds to wait between a change being requested to a property that is to be transitioned and the start of the transition effect. The default value is `0`.   
+- `needLayout`(boolean)：Whether or not the layout animation occurs when animation is executed，default value is `false`
+- `timingFunction` (string): Used to describe how the intermediate values of the styles being affected by a transition effect are calculated, default value is `linear`, the allowed attributes are listed in the following table:    
 
+| name                           | description                              |
+| :----------------------------- | :--------------------------------------- |
+| `linear`                       | Specifies a transition effect with the same speed from start to end |
+| `ease`                         | Specifies a transition effect with a slower and slower speed |
+| `ease-in`                      | Specifies a transition effect with a slow start |
+| `ease-out`                     | Specifies a transition effect with a slow end |
+| `ease-in-out`                  | Specifies a transition effect with a slow start and end |
+| `cubic-bezier(x1, y1, x2, y2)` | Define your own values in the cubic-bezier function. Possible values are parameter values from 0 to 1. More information about cubic-bezier please visit [cubic-bezier](http://cubic-bezier.com/) and [Bézier curve](https://en.wikipedia.org/wiki/B%C3%A9zier_curve). |
 
-### Example
+### ``callback``
 
-```html
-<template>
-  <div class="wrapper">
-    <div ref="test" @click="move" class="box"></div>
-  </div>
-</template>
-
-<script>
-  const animation = weex.requireModule('animation')
-  const modal = weex.requireModule('modal')
-
-  export default {
-    methods: {
-      move () {
-        var testEl = this.$refs.test;
-        animation.transition(testEl, {
-          styles: {
-            color: '#FF0000',
-            transform: 'translate(250px, 100px)',
-            transformOrigin: 'center center'
-          },
-          duration: 800, //ms
-          timingFunction: 'ease',
-          needLayout:false,
-          delay: 0 //ms
-        }, function () {
-          modal.toast({ message: 'animation finished.' })
-        })
-      }
-    }
-  }
-</script>
-
-<style scoped>
-  .box {
-    width: 250px;
-    height: 250px;
-    background-color: #DDD;
-  }
-</style>
-```
+Callback which is a function called after the completion of animation. In iOS platform, you can use function to get animation processing's information.
```

Comment:
What do you mean by saying get animation processing's information, is there any code examples ?